docs/openapi/paths/registrations.yml

```diff
@@ -6,18 +6,17 @@ post:
   summary: Register with email
   description: |-
     Register a new account with email.
-
     The backend system will send an email to the user to create a new account on the website.
+
   requestBody:
     required: true
     content:
       application/json:
         schema:
-          $ref: '../../schemas.yml#/requests_registrations_post_create'
+          $ref: '../schemas.yml#/requests_registrations_post_create'
```

Comment:
Can we use `post` or `create` only?
For me, I prefer `requests_registrations_post` as we are using the method name in the top.

Reply:
Good catch! Let me refactor in schemas folder

Reply:
Improve in this commit: https://github.com/nimblehq/nimble-survey-web/pull/95/commits/099fd28cdb43f0f480d5aa71c229a73d81b669db